examples/kubernetes/static_provisioning/README.md

```diff
@@ -18,26 +20,54 @@ spec:
   persistentVolumeReclaimPolicy: Recycle
   csi:
     driver: fsx.csi.aws.com
+    # mount root of filesystem
     volumeHandle: [FileSystemId]
+    # mount subdir of filesystem
+    # volumeHandle: [DNSName:MountName:basedir:subdir:uuid]
     volumeAttributes:
-      dnsname: [DNSName] 
+      dnsname: [DNSName]
       mountname: [MountName]
+      # set the basedir and subdir if you wanted to mount subdir
```

Comment:
in kubernetes the volumehandle must uniquely identify the volume. if you try to mount two PVC+PV on the same node and they have the same volumehandle but different volumeattributes.subdir, I don't think it works. this is why in EFS we removed the path volumeattribute and stuffed everything into the volumehandle like [FileSystemId]:[Path]  instead. https://github.com/kubernetes/kubernetes/blob/497e0dfd33b180fcc3855aad794d87f0e429c82c/pkg/volume/csi/csi_plugin.go#L316